core/src/test/java/org/bitcoinj/core/Base58DecodeCheckedInvalidChecksum.java

```diff
@@ -0,0 +1,11 @@
+package org.bitcoinj.core;
```

Comment:
All the new files are missing the license. You can use the "Copyright by the original author or authors." copyright line if you want.

Reply:
I added the same license as in the other test files.